dialer_test.go

```diff
@@ -138,6 +138,7 @@ func TestDialWithAdminAPIErrors(t *testing.T) {
 
 func TestDialWithConfigurationErrors(t *testing.T) {
 	inst := mock.NewFakeCSQLInstance("my-project", "my-region", "my-instance")
+	inst.Cert.NotAfter = time.Now().Add(-time.Hour)
```

Comment:
nit: Should this be an option on `inst`? `WithCertExpiry` or something?

Reply:
I've done exactly that in the next PR.